tiledb/sm/serialization/tiledb-rest.capnp

```diff
@@ -670,6 +670,16 @@ struct Delete {
   # Stats object
 }
 
+struct ResultTileId {
```

Comment:
This looks unused or am I missing something? If yes, let's just revert all capnp changes?

Reply:
Ack, I thought I had done so. Good catch!

Reply:
03c540b12586